lib/node_modules/@stdlib/math/base/special/ellipj/lib/am.js

```diff
@@ -38,25 +38,22 @@ var tmp = [ 0.0, 0.0, 0.0, 0.0 ];
 * @returns {number} Jacobi amplitude, `am`
 *
 * @example
+* var am = require( '@stdlib/math/base/special/ellipj' ).am;
```

Comment:
Why were these changes made?

We require package itself only in `lib/index.js`; my expected change here would have been to simply add the missing `var`s to the individual `@example`s.